kaggle_environments/envs/open_spiel/open_spiel.py

```diff
@@ -176,16 +188,25 @@ def interpreter(
     return kaggle_state
 
   # --- Get and maybe initialize game and state on the env object ---
-  if not hasattr(env, 'os_game'):
+  if not hasattr(env, "os_game"):
     game_string = env.configuration.get("openSpielGameString")
     env.os_game = pyspiel.load_game(game_string)
-  if not hasattr(env, 'os_state'):
+  if not hasattr(env, "os_state"):
+    assert hasattr(env, "os_game")
     env.os_state = env.os_game.new_initial_state()
+  assert hasattr(env, "os_game")
+  assert hasattr(env, "os_state")
   if "stateHistory" not in env.info:
-    env.info['stateHistory'] = [str(env.os_state)]
-    env.info['actionHistory'] = []
-    env.info['moveDurations'] = []
-  
+    env.info["stateHistory"] = [str(env.os_state)]
+    env.info["actionHistory"] = []
+    env.info["moveDurations"] = []
+    initial_actions = env.configuration.get("initialActions", [])
```

Comment:
can we use the seed to select the initial actions vs putting them in the config?

For example load all for your `.jsonl` and `% config.seed` to find the initial action that we want to start with.

This operation could be done on a `configuration.randomStart` flag or something similar.

That way we can start two games at the same time (with seeds 1 and 2) to select two different start positions.

Currently we *can* change seeds between games, but not configuration.